pkg/network/http/http_statkeeper.go

```diff
@@ -51,7 +59,13 @@ func (h *httpStatKeeper) GetAndResetAllStats() map[Key]RequestStats {
 }
 
 func (h *httpStatKeeper) add(tx httpTX) {
-	key := h.newKey(tx)
+	path, rejected := h.processHTTPPath(tx)
+	if rejected {
+		atomic.AddInt64(&h.telemetry.dropped, 1)
```

Comment:
Do we want to distinguish from those that are dropped because the max size has been hit, and those that are rejected because of configured rules?

Reply:
yeah, that sounds reasonable